LOG:  database system was interrupted; last known up at 2017-06-02 13:54:00 BST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/19E56D0: wanted 24, got 0
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
LOG:  database system was interrupted; last known up at 2017-06-05 16:43:26 BST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/19E7A60: wanted 24, got 0
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
ERROR:  cached plan must not change result type
STATEMENT:  SELECT  "users".* FROM "users" WHERE "users"."id" = $1  ORDER BY "users"."id" ASC LIMIT 1
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
LOG:  using stale statistics instead of current ones because stats collector is not responding
FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 768) running in data directory "/usr/local/var/postgres"?
LOG:  database system was interrupted; last known up at 2017-06-18 05:38:00 BST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/1AAB478: wanted 24, got 0
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
ERROR:  cached plan must not change result type
STATEMENT:  SELECT  "users".* FROM "users" WHERE "users"."id" = $1  ORDER BY "users"."id" ASC LIMIT 1
LOG:  database system was interrupted; last known up at 2017-06-28 21:29:45 BST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/1C74DC0: wanted 24, got 0
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  type "reference" does not exist at character 42
STATEMENT:  ALTER TABLE "consultations" ADD "doctor" reference
FATAL:  database "simple_messages_app_development" does not exist
ERROR:  insert or update on table "messages" violates foreign key constraint "fk_rails_273a25a7a6"
DETAIL:  Key (user_id)=(5) is not present in table "users".
STATEMENT:  INSERT INTO "messages" ("body", "user_id", "conversation_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4, $5) RETURNING "id"
LOG:  autovacuum launcher shutting down
LOG:  received smart shutdown request
LOG:  shutting down
LOG:  database system is shut down
LOG:  database system was shut down at 2017-07-03 19:10:05 BST
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
ERROR:  insert or update on table "messages" violates foreign key constraint "fk_rails_273a25a7a6"
DETAIL:  Key (user_id)=(5) is not present in table "users".
STATEMENT:  INSERT INTO "messages" ("body", "user_id", "conversation_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4, $5) RETURNING "id"
ERROR:  insert or update on table "messages" violates foreign key constraint "fk_rails_273a25a7a6"
DETAIL:  Key (user_id)=(5) is not present in table "users".
STATEMENT:  INSERT INTO "messages" ("body", "user_id", "conversation_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4, $5) RETURNING "id"
